DeepNeuralNet4e.py

```diff
@@ -344,3 +352,36 @@ def predict(example):
         return layer_out.index(max(layer_out))
 
     return predict
+
+
+def simple_rnn_learner(train_data, val_data, epochs=2):
+
```

Comment:
It would be great if you could add some description of the function, detailing what it does (is it just a general network?), what is the format of the data, etc. Right now it's hard to discern what this is trying to accomplish.

Reply:
Thanks for the reminder! I will also check other functions for the necessary comments.

Reply:
@bruceyoungsysu : Hi Bruce. There is one failing continuous integration test below. Can you fix this before I approve your pull request. I've attached a screen shot of the error.
![Failing-Test-DeepNN](https://user-images.githubusercontent.com/1411402/60397512-5bb73b00-9b4e-11e9-9b99-913dbd2868b2.jpg)